packages/relay/src/lib/constants.ts

```diff
@@ -18,6 +18,8 @@
   *
   */
 
+import {PrecheckStatusError} from "@hashgraph/sdk";
```

Comment:
Q: where is this used?

Reply:
in the catch, for new condition `if(e instanceof PrecheckStatusError) {`

Reply:
There's no catch in this file.

Reply:
ohhh, I see it now, hahaha, removing it now. not sure how it got there, thanks for noticing!! 👍

Reply:
✅  Done